gen/final.ml

```diff
@@ -186,6 +186,12 @@ module Make : functor (O:Config) -> functor (C:ArchRun.S) ->
                 Some (S (Code.add_tag (Code.as_data evt.C.C.loc) evt.C.C.v))
             | Code.Pte ->
                 Some (P evt.C.C.pte)
+            | Code.Instr ->
+              let s = match evt.C.C.ins with
+              | 1 -> "instr:\"NOP\""
+              | 0 -> "instr:\"B .+12\""
```

Comment:
Another problem with checking comparing a register value with a branch is that this AArch64 specific. I think it would be simpler to check if is a nop or not.

Reply:
Yes, I have made this to be architecture dependent in the newest commit. Let me know if what I did is unclear